runtime/near-vm-logic/src/alt_bn128.rs

```diff
@@ -27,25 +27,23 @@ pub fn alt_bn128_g1_multiexp_sublinear_complexity_estimate(
     const MULTIEXP_ITEM_SIZE: u64 = std::mem::size_of::<(G1, Fr)>() as u64;
 
     // A+C*n/(log2(n)+4) - B*n - discount
-
-    let n = (n_bytes - 1).checked_add(MULTIEXP_ITEM_SIZE).ok_or(HostError::IntegerOverflow)?
```

Comment:
Note: I don't fully understand the change from `(n_bytes + size - 1) / size` to `(n_bytes + 2*size) / size`. The former compute ceil divion, the latter feels like bug.